app/views/shared/_citations.html.erb

```diff
@@ -19,7 +19,7 @@
   <% @presenter.creator.each do |creator| %>
   <meta name="citation_author" content="<%= creator %>"/>
   <% end %>
-  <meta name="citation_publication_date" content="<%= @presenter.date_created.first %>"/>
+  <meta name="citation_publication_date" content="<%= @presenter.date_created.first unless @presenter.date_created.nil? %>"/>
```

Comment:
Do we need to have a conditional here?  Can we update the presenter instead such that it always returns an array?